web_external/templates/widgets/wmsFeatureInfoWidget.jade

```diff
@@ -0,0 +1,6 @@
+div(id='wms_info_dialog', title="WMS Layer Info")
```

Comment:
I'd like us to start namespacing using this convention `m-wms-info-dialog`.

There are 3 uses of this in WMSFeatureInfoWidget.js also.